assets/less/zenodo-rdm/globals/site.overrides

```diff
@@ -22,6 +22,19 @@ body {
   }
 }
 
+.mce-content {
```

Comment:
I would move this to invenio-theme instead, since it affecting the whole core, not only zenodo